rpc_jobs/osa_sha_update.yml

```diff
@@ -0,0 +1,92 @@
+- job:
+    name: 'OSA-SHA-Update-PR'
+    project-type: workflow
+    logrotate:
+      daysToKeep: 14
+    parameters:
+        - string:
+            name: OSA_REPO
+            default: "https://github.com/openstack/openstack-ansible"
+        - string:
+            name: OSA_BRANCH
+            default: "stable/newton"
+        - string:
+            name: "RPC_REPO"
+            default: "https://github.com/rcbops/rpc-openstack"
+        - string:
+            name: "RPC_BRANCH"
+            default: "master"
+        - rpc_gating_params
+    dsl: |
+      node(){
+        deleteDir()
```

Comment:
So this here is problematic -- I guess we'll need to remove this or have osa-differ just write outside of the workspace?